Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.23-openshift-4.19 AS builder
+WORKDIR /go/src/github.com/openshift/volume-data-source-validator
+COPY . .
+RUN make build
+
+FROM registry.ci.openshift.org/ocp/4.19:base-rhel9
+COPY --from=builder /go/src/github.com/openshift/volume-data-source-validator/bin/volume-data-source-validator /usr/bin/
+ENTRYPOINT ["/usr/bin/volume-data-source-validator"]
```

Comment:
Would it make sense to rename this file to `Dockerfile.openshift`? In part for consistency, but also in case upstream ever adds `Dockerfile` it would avoid merge conflicts.

Reply:
Good point, I kind of used azcopy repo I worked on recently as a template (which has the same mistake :/) https://github.com/openshift/azure-storage-azcopy/blob/main/Dockerfile

Renaming.